src/components/GameView/GameOverDialog.vue

```diff
@@ -1,26 +1,43 @@
 <template>
   <v-dialog v-model="show" persistent max-width="650">
     <v-card id="game-over-dialog">
-      <v-card-title :data-cy="headingDataAttr">
-        <h1>{{ heading }}</h1>
-      </v-card-title>
-      <v-card-text class="d-flex justify-center">
-        <v-img
-          :src="logoSrc"
-          :data-cy="logoDataAttr"
-          class="logo-image"
-        />
```

Comment:
Let's make the non-ranked dialog look exactly as it does now, without changes (big img in the center) since the match stuff doesn't matter in that context and we may as well continue to use the same real estate. If it's easier to manage, we could split this into two components to handle the ranked and unranked cases and then include both in a higher order component that chooses which dialog to show based on ranked/unranked

Reply:
I agree with this. and if you need to split this I think it should be a follow up as long as you can reasonably finish this without making too many forks in the template.